data/themes/darktable.css

```diff
@@ -1527,35 +1527,25 @@ margin: 0 2px;
   padding: 0.45em;
 }
 
-#basics-box,
-#basics-box-labels
-{
-  padding-bottom: 0.75em;
-}
-
 #basics-box
 {
+  padding-bottom: 0.75em;
   padding-top: 0.5em;
 }
 
 #basics-box #module-enable-button,
-#basics-box-labels #module-enable-button
-{
-  min-height: 1em;
-  min-width: 1em;
-  border: none;
-}
-
+#basics-box-labels #module-enable-button,
 #basics-link
 {
-  min-height: 1em;
-  min-width: 1em;
-  margin: 0 0 0.3em 0.75em;
+  min-height: 0.95em;
+  min-width: 0.59em;
```

Comment:
whoops - typo?

Reply:
Just see that and already fixed! Thanks!